src/sentry/static/sentry/app/views/settings/components/forms/tableField.tsx

```diff
@@ -126,12 +110,26 @@ export default class TableField extends React.Component<Props> {
       return <div>{button}</div>;
     }
 
+    const renderConfirmMessage = () => {
+      return (
+        <React.Fragment>
+          <Alert type="error">
+            <span
+              dangerouslySetInnerHTML={{
+                __html: singleLineRenderer(confirmDeleteMessage || ''),
+              }}
+            />
+          </Alert>
```

Comment:
Is an empty alert going to look ok in the confirm box?

Reply:
@markstory Good point, I put in a default message